server/server.go

```diff
@@ -16,19 +17,24 @@ import (
 func upload(w http.ResponseWriter, r *http.Request) {
 	fmt.Println("File Upload Endpoint Hit")
 
-	w.Header().Add("Content-Type", "application/json")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+
 	responseString := readFormData(r)
 
+	log.Println(responseString)
```

Comment:
why is the response string being printed?

Reply:
I was testing something. But will remove it